softwarereview_intro.pt.Rmd

```diff
@@ -0,0 +1,86 @@
+- A revisão é uma conversa de mão dupla. Ao revisar os pacotes, você terá a chance de **continuar aprendendo práticas de desenvolvimento com os autores e outros revisores**.
+- A natureza aberta do nosso processo de revisão permite que você **você faça contatos e conheça colegas e colaboradores** durante o processo de revisão. Nossa comunidade é amigável e está repleta de membros que dão suporte a você, especialistas em desenvolvimento de R e em muitas outras áreas da ciência e da computação científica.
+- Para se voluntariar para ser um de nossos revisores, preencha o formulário [este breve formulário](https://airtable.com/shrnfDI2S9uuyxtDw) fornecendo suas informações de contato e áreas de especialização. Estamos sempre procurando mais revisores com experiência geral em redação de pacotes e com conhecimento especializado nas áreas em que os pacotes são usados.
+
+## Por que as avaliações estão abertas? {#whyopen}
+
+Nossos tópicos de avaliação são públicos. Autores, avaliadores e editores conhecem as identidades uns dos outros. A comunidade em geral pode ver ou até mesmo participar da conversa enquanto ela acontece. Isso incentiva você a ser minucioso e a fazer revisões construtivas e não contraditórias. Tanto os autores quanto os [revisores relatam](https://ropensci.org/tags/reviewer/) que gostam e aprendem mais com essa troca aberta e direta. Isso também traz o benefício de criar uma comunidade. Os participantes têm a oportunidade de se relacionar de forma significativa com novos colegas, e novas colaborações surgiram por meio de ideias geradas durante o processo de revisão.
+
+Estamos cientes de que os sistemas abertos podem ter desvantagens. Por exemplo, na revisão acadêmica tradicional, [a revisão por pares duplo-cega pode aumentar a representação de autores do sexo feminino](https://www.sciencedirect.com/science/article/pii/S0169534707002704) sugerindo preconceito em revisões não cegas. Também é possível que os revisores sejam menos críticos na revisão aberta. No entanto, afirmamos que a abertura da conversa sobre a revisão proporciona uma verificação da qualidade e da parcialidade da revisão; é mais difícil injetar comentários subjetivos ou sem embasamento em público e sem a cobertura do anonimato. Em última análise, acreditamos que a comunicação direta e pública entre autores e revisores melhora a qualidade e a imparcialidade das revisões.
+
+Além disso, os autores e revisores podem entrar em contato com os editores em particular se tiverem alguma dúvida ou pergunta.
+
+## Como os usuários saberão que um pacote foi revisado? {#how-will-users-know-a-package-has-been-reviewed}
+
+- O LEIAME do seu pacote apresentará um selo de revisão por pares com um link para o tópico de revisão do software.
+- Seu pacote receberá um selo [`docs.ropensci.org` site de documentos](#rodocsci) que você pode vincular a partir da DESCRIÇÃO.
+- O repositório do seu pacote será transferido para a organização rOpenSci.
```

Comment:
"No GitHub" was an addition to make it clearer, but it's not in the english version of the book, so it's good to check with @yabellini if it is okay to make this change.
```suggestion
- Se você quiser, o repositório do seu pacote poderá ser transferido para a organização rOpenSci no GitHub.
```

Reply:
Yes, it is ok to change the text from English to make the Portuguese version clearer.  We have done that in the Spanish version too.